float64[] q
float64[] dq
float64[] ee_translation
float64 theta_z
float64 f_ext
